src/main/resources/versions.json

```
@@ -1,7 +1,7 @@
 {
     "supportedVersions": {
-        "8.4.0-alpha02": [
-            "8.5"
+        "8.4.0-alpha07": [
+            "8.6-rc-1"
```

```suggestion
            "8.6-rc-3"
```
Why use rc 1 if rc 3 is out?

---

Autopilot 🤦